pcdsdevices/pneumatic.py

```diff
@@ -0,0 +1,69 @@
+
+"""
+Pneumatic Classes.
+
+This Module contains all the classes relating to Pneumatic Actuators
+"""
+
+from lightpath import LightpathState
+from ophyd import Component as Cpt
+from ophyd import EpicsSignal, EpicsSignalRO
+
+from pcdsdevices.interface import LightpathMixin
```

Comment:
import for the interface utilities
```suggestion
from pcdsdevices.interface import BaseInterface, LightpathMixin
```

Reply:
done.